app/src/main/java/com/android/solvit/seeker/ui/provider/ProviderInfo.kt

```diff
@@ -0,0 +1,401 @@
+            onClick = { Toast.makeText(context, "Not implemented", Toast.LENGTH_SHORT).show() },
+            modifier = Modifier.testTag("menuButton")) {
+              Icon(
+                  painter = painterResource(id = R.drawable.menu_icon),
+                  contentDescription = "Menu",
+                  modifier = Modifier.size(24.dp),
+                  tint = Color.Unspecified)
+            }
+      }
+}
+
+@Composable
+fun ProviderHeader(provider: Provider) {
+  Box(modifier = Modifier.fillMaxWidth().background(Color.White).testTag("providerHeader")) {
+    Row(
+        modifier = Modifier.fillMaxWidth().padding(16.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween) {
+          val context = LocalContext.current
+          Row(verticalAlignment = Alignment.CenterVertically) {
+            AsyncImage(
+                model = if (provider.imageUrl != "") provider.imageUrl else R.drawable.default_pdp,
+                contentDescription = "Profile Picture",
+                contentScale = ContentScale.Crop,
+                modifier =
+                    Modifier.size(128.dp)
+                        .border(2.dp, Color.Transparent, RoundedCornerShape(16.dp))
+                        .clip(RoundedCornerShape(16.dp))
+                        .testTag("providerImage"))
+            Spacer(modifier = Modifier.width(16.dp))
+            Column {
+              Text(
+                  text = provider.name,
+                  fontSize = 20.sp,
+                  fontWeight = FontWeight.Bold,
+                  modifier = Modifier.testTag("providerName"))
+              Text(
+                  text = provider.companyName,
+                  color = Color.Gray,
+                  modifier = Modifier.testTag("providerCompanyName"))
+            }
+          }
+
+          // Share icon on the right
+          IconButton(
+              onClick = { Toast.makeText(context, "Not implemented", Toast.LENGTH_SHORT).show() },
+              modifier = Modifier.testTag("shareButton")) {
+                Icon(Icons.Default.Share, contentDescription = "Share")
+              }
+        }
+  }
+}
+
+@Composable
+fun ProviderTabs(selectedTabIndex: Int, onTabSelected: (Int) -> Unit) {
+  TabRow(
+      selectedTabIndex = selectedTabIndex,
+      modifier = Modifier.fillMaxWidth().testTag("providerTabs"),
+      containerColor = Color(0xFF0099FF),
+      contentColor = Color.White) {
+        Tab(
+            selected = selectedTabIndex == 0,
+            onClick = { onTabSelected(0) },
+            modifier = Modifier.testTag("profileTab")) {
+              Text("Profile", modifier = Modifier.padding(16.dp))
+            }
+        Tab(
+            selected = selectedTabIndex == 1,
+            onClick = { onTabSelected(1) },
+            modifier = Modifier.testTag("reviewsTab")) {
+              Text("Reviews", modifier = Modifier.padding(16.dp))
+            }
+      }
+}
```

Comment:
Nit: The selected tab is not highlighted in the same color as in the Figma and is not very noticeable

Reply:
Thanks for noticing! I do not think it is a really important difference for now and frankly I did not see an option to modify it.